### ISSUE TITLE
The CI server isn't completing package builds

### ISSUE THREAD

**Issue by deanmarcussen** [OrchardCMS/OrchardCore#7006]
It seems the CI server isn't completing for the last six or so builds.

Failed : https://ci.appveyor.com/project/alexbocharov/orchardcore/builds/34987382

Last passed : https://ci.appveyor.com/project/alexbocharov/orchardcore/builds/34902901

Not sure why but it's dropping at the same point every time.

**Comment by jtkech** [comment]
Okay, i think i found the issue, the 5 packages that are not built with the current sdk version are the `OC.*.Targets` that doesn't have any source files. Maybe we just need to remove `IncludeBuildOutput = false` in their project files, and / or add to them at least one csharp source file, i will try it tomorrow

**Comment by deanmarcussen** [comment]
Interesting @jtkech 

The issue occurs since we went up to the latest sdk release
```
dotnet --version
3.1.401
```

Last good build was
```
dotnet --version
3.1.302
```

Locally for me your fix removes almost all the errors, but I still get one last error trying to pack the `OpenId` module
```
/usr/local/share/dotnet/sdk/3.1.401/Sdks/NuGet.Build.Tasks.Pack/buildCrossTargeting/NuGet.Build.Tasks.Pack.targets(198,5): error NU5129: - At least one .props file was found in 'build//', but 'build/OrchardCore.OpenId.props' was not. [/Users/deanmarcussen/Projects/OrchardCore/src/OrchardCore.Modules/OrchardCore.OpenId/OrchardCore.OpenId.csproj]
/usr/local/share/dotnet/sdk/3.1.401/Sdks/NuGet.Build.Tasks.Pack/buildCrossTargeting/NuGet.Build.Tasks.Pack.targets(198,5): error NU5129:  [/Users/deanmarcussen/Projects/OrchardCore/src/OrchardCore.Modules/OrchardCore.OpenId/OrchardCore.OpenId.csproj]
```

I noticed as well when I took the 3.1.401 upgrade that the MSBuild version has gone to a preview version.
`16.7.0-preview-20360`

**Comment by jtkech** [comment]
Fixed through #7019 ;)
